lib/rubocop/cop/graphql/ordered_arguments.rb

```diff
@@ -80,16 +97,26 @@ def register_offense(previous, current)
         end
 
         def argument_name(node)
-          node.first_argument.value.to_s
+          argument = node.block_type? ? node.children.first.first_argument : node.first_argument.value.to_s
```

Comment:
this part — `node.children.first.first_argument` is probably quite brittle

Reply:
that's fine—we have specs 🙂